playbooks/templates/.env.j2

```diff
@@ -176,19 +176,20 @@ SKYNET_ACCOUNTS_LOG_LEVEL={{ webportal_common_config.accounts_log_level }}
 # - 'false': All users incl. anonymous users can access the portal (access to portal is public)
 # REQUIRED: manually defined by user, or default is set in docker-compose file
 ACCOUNTS_LIMIT_ACCESS={{ webportal_common_config.accounts_limit_access | string | lower }}
+{% endif %}
 
-{% if webportal_common_config.accounts_limit_access | string | lower != 'false' %}
-# Account Test account for health checks.
+{% if webportal_common_config.accounts_limit_access | default('') | string | lower != 'false' %}
+# Account Test account for health checks and integration tests.
 #
 # REQUIRED when ACCOUNTS_LIMIT_ACCESS is not 'false': manually defined by user.
 # API key takes precedence over email and password. It is recommended.
-# Email and password are deprecated and will be removed in the future.
+# Email and password are deprecated in use for the health checks but are needed
+# for generating the API key.
 ACCOUNTS_TEST_USER_API_KEY={{ webportal_server_config.accounts_test_user_api_key }}
 ACCOUNTS_TEST_USER_EMAIL={{ webportal_server_config.accounts_test_user_email }}
 ACCOUNTS_TEST_USER_PASSWORD={{ webportal_server_config.accounts_test_user_password }}
-
-{% endif %}
 {% endif %}
+
```

Comment:
Keep the original formatting, otherwise if `if` condition is false, double empty line is rendered.

Reply:
I don't understand why a space after the if instead of a space before the if matters here. At the end of the day it is one space. If a space is added when the if is removed, there would be two spaces in both cases right?

Also, formatting lint this feels like a `nit` or at least a follow up and shouldn't be a blocking comment.

Reply:
This is because conditional blocks in this template are done this way:

```
some-row-1

{if}
optional-row-2

{endif}
some-row-3
```

When condition is true:
```
some row-1

optional-row-2

some-row-3
```

When condition is false and empty row after the block is within if block:
```
some-row-1

some-row-3
```

When condition is false and empty row after the block is outside if block:
```
some-row-1


some-row-3
```

Reply:
nit